client/src/components/App/Content/LandingPage/InvestigationTable/SettingsActions/SettingsActions.tsx

```diff
@@ -0,0 +1,73 @@
+import React, { useState } from 'react';
```

Comment:
export every onClick function outside (declare the function outside the return and use it inside)

Reply:
fixed